content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
+
+For more information, see [the React Native Proguard documentation][13].
+
+If you are still unsure, you can see if running `(cd android && ./gradlew tasks --all) | grep minifyReleaseWithR8` returns anything. If so, minification is enabled.
+
+{{% collapse-content title="Manually on each build" level="h5" %}}
+
+In your `android/app/build.gradle` file, add the [latest version of the plugin][14] and configure it **at the top of the file**:
+
+```groovy
+plugins {
+    id("com.datadoghq.dd-sdk-android-gradle-plugin") version "x.y.z"
+}
+
+datadog {
+    checkProjectDependencies = "none" // this is needed in any case for React Native projects
+}
+```
+
+For the upload to work, you need to provide your Datadog API key. You can specify it as a `DATADOG_API_KEY` environment variable, or create a `datadog-ci.json` file at the root of your project containing the API key:
+
+```json
+{
+    "apiKey": "<YOUR_DATADOG_API_KEY>"
+}
+```
+
+You can also specify the Datadog site (such as `datadoghq.eu`) as a `DATADOG_SITE` environment variable, or as a `datadogSite` key in your `datadog-ci.json` file.
+For more information, see the [Datadog Android SDK Gradle Plugin][15].
+
+To run the plugin after a build run `(cd android && ./gradlew app:uploadMappingRelease)`.
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Automate the upload on each build" level="h5" %}}
+
+Install the plugin like in the previous step.
+
+Find the loop on `applicationVariants` in the `android/app/build.gradle` file. It should look like `applicationVariants.all { variant ->`.
+
+Inside the loop, add the following snippet:
+
+```groovy
+        if (project.tasks.findByName("minify${variant.name.capitalize()}WithR8")) {
+            tasks["minify${variant.name.capitalize()}WithR8"].finalizedBy { tasks["uploadMapping${variant.name.capitalize()}"] }
+        }
+```
+
+**Note**: Re-uploading a source map does not override the existing one if the version has not changed.
+
+{{% /collapse-content %}}
+
+
+## Troubleshooting
+
+### Usage with `use_frameworks!`
+
+If you have `use_frameworks!` enabled in your `Podfile`, running `pod install` after adding the SDK is likely to trigger an error like this one:
+
+```shell
+The 'Pods-MyApp' target has transitive dependencies that include statically linked binaries: (DatadogSDKBridge, DatadogSDKCrashReporting)
+```
+
+To prevent that error, edit your `Podfile` to install the React Native SDK pod as a static library:
+
+```ruby
+static_libraries = ['DatadogSDKReactNative']
+
+# Turn pods with static dependencies into static libraries by overriding the static_framework? function to return true
+pre_install do |installer|
+  installer.pod_targets.each do |pod|
+    if static_libraries.include?(pod.name)
+      def pod.static_framework?;
+        true
+      end
+      def pod.build_type;
+        Pod::BuildType.static_library
+      end
+    end
+  end
+end
+```
+
+**Note**: This solution comes from this [StackOverflow][8] post.
```

Comment:
This isn't opening to a StackOverflow post, and I'm not seeing one in the list of reference links